chat-plugins/modlog.js

```diff
@@ -280,15 +318,30 @@ exports.commands = {
 		let addModlogLinks = Config.modloglink && (!hideIps || (targetRoom && targetRoom.isPrivate !== true));
 		// Let's check the number of lines to retrieve or if it's a word instead
 		let lines = 0;
+		let more = false;
+		let moreIncrement = 0;
+		if (target.includes('LINE INCREMENT=')) {
```

Comment:
Nah, it shouldn't be documented at all, since for most people it shouldn't be necessary to specify in general.